pkg/systemd/ssh-list-host-keys.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -eu
+port="22"
+host="localhost"
+
+parse_addr() {
+    if [ -n "$1" ]; then
+        count=$(echo "$1" | grep -o ":" | wc -l)
```

Comment:
Instead of spawning yet more processs, can we make this cleaner and clearer by using:

```
case "$1" in
*:*)
     ...
     ;;
*)
     ....
     ;;
esac
```